weldx/asdf/schemas/weldx.bam.de/weldx/core/time_series-1.0.0.yaml

```diff
@@ -0,0 +1,85 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://weldx.bam.de/schemas/weldx/core/time_series-1.0.0"
+tag: "tag:weldx.bam.de:weldx/core/time_series-1.0.0"
+
+title: |
+  Schema that describes a time series.
+description: |
+  Describes a time dependent quantity.
+anyOf:
+  - type: object
+    description: |
+      Implementation for constant values.
+    properties:
+      values:
+        description: |
+          Number or n-dimensional array that is constant in time.
+        anyOf:
+          - type: number
+          - tag: "tag:stsci.edu:asdf/core/ndarray-1.0.0"
+      unit:
+        description: |
+          Unit of the data.
+        type: string
+    required: [values, unit]
+    additionalProperties: false
+
+  - type: object
+    description: |
+      Implementation for expressions.
+    properties:
+      values:
+        description: |
+          A mathematical expression that describes the time dependent behaviour.
+        tag: "tag:weldx.bam.de:weldx/core/mathematical_expression-1.0.0"
+      unit:
+        description: |
+          Resulting unit of the data when the expression is evaluated.
+        type: string
+      shape:
+        description: |
+          (optional) Resulting shape of the data when the expression is evaluated.
+        type: array
+      required: [values, unit]
+      additionalProperties: false
```

Comment:
decreasing indent by one level (currently listed as property)

Reply:
Did the trick.